### ISSUE TITLE
The local state is not update after click back button

### ISSUE THREAD

**Issue by Franco98404** [FlutterFlow/flutterflow-issues#200]
Issue tracker is **ONLY** used for reporting bugs. New feature suggestions and questions should be discussed on Community or submitted through our user feedback form.

Your issue may already be reported! Please search in the [issue tracker](../) before creating one.

Please **thumbs up** this issue if you have also experienced it. You may also add more information if there is something relevant that was not mentioned. However, please refrain from comments that are not constructive, like "I have this problem too", etc.

## Expected behavior (required)

The local state and the fill color of the side panel should be updated

## Current behavior (required)

The local state is not updated therefore the fill color is not changed

## To Reproduce (required)

1. Click on the Scaffold and then click on Action Flow Editor
2. Update the local state
3. Click run
4. Go to another page
5. Click back button
6. The fill color is not update because the local state is not update

## Context (required)

I want to change the fill color of the container

## Screenshots / recordings

https://schoolfy365.tinytake.com/msc/NzY1MTEwMl8yMDY0MTUzMA

## Your environment

* Version of FlutterFlow used: November 18, 2022 - Flutter 3.3.4
* Platform (e.g. Web, MacOS Desktop): Android, Desktop Web and Mobile Web
* Browser name and version: Chrome 107.0.5304.141
* Operating system and version (desktop or mobile): Android 12 and Web

**Comment by Franco98404** [comment]
Any news about this problem? @agreaves @bulgariamitko @wenkaifan0720

**Comment by bulgariamitko** [comment]
I believe the color is not changed, because when I user goes back it is only showing the state of the app that was at the moment the page was first visited. If you like to update anything on the page you should not use the back button and instead you need to navigate to that page. Try that and let me know if that will work.

**Comment by Franco98404** [comment]
If I add the navigation to the page probably will solve the problem but in the mobile app will persist the problem, because if the user use the back button that have integrated in OS will have the same problem

**Comment by Franco98404** [comment]
Analyzing the problem I think that I can change the title to a more descriptive like "On page load action is not working using the back button", what do you think?

**Comment by bulgariamitko** [comment]
It will never work if you click the back button as the page is not loaded it is just showing the same page that was already loaded. At least this is my knowledge of Flutter. You can disable Allow back Navigation when redirecting to the same page and this will not case the user to go in a loop.

**Comment by Franco98404** [comment]
It's disabled the back navigation but always when press the back button goes to the "Logged In" page and doesn't load the page action

**Comment by bulgariamitko** [comment]
I think this is not a bug, but I will let FF team decide.

**Comment by shelpert** [comment]
Yes, it is not a bug that back navigation does not reload the page and therefore does not update the local state. However, it is half bug/half feature request that disabling back navigation does not disable the built-in Android back button. 

I'm going to close this issue but raise the Android back navigation issue with the team so we can discuss the best way to solve it.

**Comment by Franco98404** [comment]
Ok, I understand but is there a way to stay updated about this when the FF team decide? @shelpert @bulgariamitko
